src/main/java/seedu/address/model/client/Buyer.java

```diff
@@ -70,4 +70,13 @@ public String toString() {
                 .add("email", email)
                 .toString();
     }
+
```

Comment:
I think it's OK to have the isBuyer and isSeller methods, but we could omit it to and check if an object is a buyer/ seller using `objx instanceof Buyer`/ `objectx instanceof Seller`